Code/Framework/AzToolsFramework/Tests/EditorTransformComponentSelectionTests.cpp

```diff
@@ -835,6 +964,32 @@ namespace UnitTest
         EXPECT_TRUE(selectedEntitiesAfter.empty());
     }
 
+    TEST_F(EditorTransformComponentSelectionViewportPickingManipulatorTestFixture, StickyDoubleClickWithSelectionWillDeselectEntities)
+    {
+        AzToolsFramework::ed_viewportStickySelect = true;
+
+        PositionEntities();
+        PositionCamera(m_cameraState);
+
+        AzToolsFramework::SelectEntities({ m_entityId1, m_entityId2, m_entityId3 });
+
+        using ::testing::UnorderedElementsAre;
+        auto selectedEntitiesBefore = SelectedEntities();
+        EXPECT_THAT(selectedEntitiesBefore, UnorderedElementsAre(m_entityId1, m_entityId2, m_entityId3));
+
+        // position in space above the entity
```

Comment:
this comment is a bit confusing since there are multiple entities - would something like "position is space away from the entities" work?

Reply:
Ah yeah this could be from an earlier version when there was only one entity, I've updated to `// position in space above the entities`